common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -1,982 +1,1001 @@
+  "feature.wynntils.itemHighlight.hotbarOpacity.description": "快捷欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.hotbarOpacity.name": "快捷欄醒目提示透明度",
+  "feature.wynntils.itemHighlight.inventoryHighlightEnabled.description": "啟用物品欄中物品的醒目提示",
+  "feature.wynntils.itemHighlight.inventoryHighlightEnabled.name": "物品欄物品醒目提示",
+  "feature.wynntils.itemHighlight.inventoryOpacity.description": "物品欄中醒目提示的物品應該使用什麼層級的不透明度?",
+  "feature.wynntils.itemHighlight.inventoryOpacity.name": "物品欄醒目提示不透明度",
+  "feature.wynntils.itemHighlight.legendaryHighlightColor.description": "Legendary 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.legendaryHighlightColor.name": "傳奇物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.legendaryHighlightEnabled.description": "啟用 Legendary 物品的醒目提示",
+  "feature.wynntils.itemHighlight.legendaryHighlightEnabled.name": "傳奇物品醒目提示",
+  "feature.wynntils.itemHighlight.mythicHighlightColor.description": "Mythic 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.mythicHighlightColor.name": "神話物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.mythicHighlightEnabled.description": "啟用 Mythic 物品的醒目提示",
+  "feature.wynntils.itemHighlight.mythicHighlightEnabled.name": "神話物品醒目提示",
+  "feature.wynntils.itemHighlight.name": "物品醒目提示",
+  "feature.wynntils.itemHighlight.normalHighlightColor.description": "Normal 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.normalHighlightColor.name": "一般物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.normalHighlightEnabled.description": "啟用 Normal 物品的醒目提示",
+  "feature.wynntils.itemHighlight.normalHighlightEnabled.name": "一般物品醒目提示",
+  "feature.wynntils.itemHighlight.oneStarIngredientHighlightColor.description": "1 星素材應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.oneStarIngredientHighlightColor.name": "1 星素材醒目提示顏色",
+  "feature.wynntils.itemHighlight.oneStarIngredientHighlightEnabled.description": "啟用 1 星素材的醒目提示",
+  "feature.wynntils.itemHighlight.oneStarIngredientHighlightEnabled.name": "1 星素材醒目提示",
+  "feature.wynntils.itemHighlight.oneStarMaterialHighlightColor.description": "1 星材料應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.oneStarMaterialHighlightColor.name": "1 星材料醒目提示顏色",
+  "feature.wynntils.itemHighlight.oneStarMaterialHighlightEnabled.description": "啟用 1 星材料的醒目提示",
+  "feature.wynntils.itemHighlight.oneStarMaterialHighlightEnabled.name": "1 星材料醒目提示",
+  "feature.wynntils.itemHighlight.powderHighlightEnabled.description": "啟用粉塵的醒目提示",
+  "feature.wynntils.itemHighlight.powderHighlightEnabled.name": "粉塵醒目提示",
+  "feature.wynntils.itemHighlight.rareHighlightColor.description": "Rare 物品應該要用什麼顏色的醒目提示?",
+  "feature.wynntils.itemHighlight.rareHighlightColor.name": "稀有物品醒目提示顏色",
+  "feature.wynntils.itemHighlight.rareHighlightEnabled.description": "啟用 Rare 物品的醒目提示",
+  "feature.wynntils.itemHighlight.rareHighlightEnabled.name": "稀有物品醒目提示",
+  "feature.wynntils.itemHighlight.setHighlightColor.description": "套裝物品應該要用什麼顏色的醒目提示?",
```

Comment:
If you are doing the "unless" portion of the above comment, then `套裝` should be `set` here, same for the description below.

Reply:
I only do this in "Item Rarities",such as: `Mythic`, `Fabled`, `Legendary`, `Rare`, `Unique`, Normal`, I think `set` in Chinese is clear enough for users. 
Or do you think I should translate this?

Reply:
Also this can make the string prettier, I don't want to do something like this:
`傳奇 (Legendary) 物品醒目提示顏色`
This will make the string too long, and looks duplicate between the word `傳奇` and `Legendary`.

Reply:
Fair enough. I think you should translate `Set` for consistency purposes.